scripts/shellcheck.sh

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
-
-set -e
-
-docker run -v "$(pwd)":/wrk -w /wrk koalaman/shellcheck scripts/*.sh
```

Comment:
The benefit of having this is that anyone could run the script locally and not need to have shellcheck installed. Just docker.

Reply:
I was umm-ing and err-ing about that too, my thought process behind removing it:
* Shellcheck is a very commonly available package
* If you're working on shell scripts, you _should_ have it installed locally, or install it.
* I added it in to `zig build` as `zig build lint_shellcheck` which globally lints all shellscripts. There's nothing to say that we couldn't make that run docker, and run like this, but it doesn't feel great to me.